js/components/portal-dashboard/all-responses-popup/selected-student-list.tsx

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import { Map } from "immutable";
+import { AnonymizeStudents } from "../anonymize-students";
+import { QuestionTypes } from "../../../util/question-utils";
+import Answer from "../../../containers/portal-dashboard/answer";
+import { getFormattedStudentName } from "../../../util/student-utils";
+import striptags from "striptags";
+
+import SpotlightIcon from "../../../../img/svg-icons/spotlight-icon.svg";
+import AssignmentIcon from "../../../../img/svg-icons/assignment-icon.svg";
+import SmallCloseIcon from "../../../../img/svg-icons/small-close-icon.svg";
+
+import css from "../../../../css/portal-dashboard/all-responses-popup/selected-student-list.less";
+
+interface IProps {
+  currentActivity?: Map<string, any>;
+  currentQuestion?: Map<string, any>;
+  students?: any;
+  anonymous: boolean;
+  isAnonymous: boolean;
+  handleCloseSpotlightStudentListDialog: (show: boolean) => void;
```

Comment:
This is probably better named `onCloseDialog` so it becomes:
`onCloseDialog: (show: boolean) => void;`

Reply:
With currying in `StudentResponsePopup`, we can write it simply as: `onCloseDialog: () => void;`